manage_sections/tests/test_view_create_section_form.py

```diff
@@ -153,3 +154,16 @@ def test_section_form_view_status_without_custom_canvas_course_id(self, log_repl
         request.LTI['lis_course_offering_sourcedid'] = None
         create_section_form(request)
         render.assert_called_with(request, 'manage_sections/error.html', status=500)
+
+    @patch('manage_sections.views.logger.error') # Mock the logger to keep log messages off the console.
```

Comment:
nitpick -- two spaces before the comment as per [PEP-8](https://www.python.org/dev/peps/pep-0008/#inline-comments).